kernel/syscall.c

```diff
@@ -655,7 +690,8 @@ static long _openat(
     if (!(locals = malloc(sizeof(struct locals))))
         ERAISE(-ENOMEM);
 
-    ECHECK(myst_get_absolute_path_from_dirfd(dirfd, pathname, 0, &abspath));
+    ECHECK(myst_get_absolute_path_from_dirfd(dirfd, pathname, 0, &abspath, 0));
```

Comment:
Please use the defined macros instead of `0` as the last parameter. Same applies to the following myst_get_absolute_path_from_dirfd calls.